submit/controllers/classification.py

```diff
@@ -0,0 +1,96 @@
+"
```

Comment:
Needs triple quotes (here and subsequent files) to match.

Reply:
I fixed and pushed these to GitHub a little while ago.